monitor-agent/src/utils/validation.ts

```
@@ -0,0 +1,108 @@
+          );
+        }
+        break;
+        
+      case 'header_exists':
+        if (!result.headers[rule.header!]) {
+          errors.push(`Required header missing: ${rule.header}`);
+        }
+        break;
+        
+      case 'header_value':
+        const headerValue = result.headers[rule.header!];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)